ureg/lib/systemrdl/src/lib.rs

```diff
@@ -473,3 +456,20 @@ pub fn translate_addrmap(addrmap: systemrdl::ParentScope) -> Result<Vec<Register
     }
     Ok(blocks)
 }
+
+#[cfg(test)]
```

Comment:
I think we can actually delete the usage of `next_multiple_of` now that we have the builtin method on integer types.

Reply:
This change just happens via cargo fmt. I think functional differences are best left to a separate PR?